osmosis-1/osmosis.zone_assets.json

```diff
@@ -1569,6 +1569,16 @@
       "chain_name": "osmosis",
       "base_denom": "factory/osmo1s794h9rxggytja3a4pmwul53u98k06zy2qtrdvjnfuxruh7s8yjs6cyxgd/umbrn",
       "osmosis_verified": true
+    },
+    {
+      "chain_name": "stargaze",
+      "base_denom": "factory/stars16da2uus9zrsy83h23ur42v3lglg5rmyrpqnju4/dust",
+      "path": "transfer/channel-75/factory/stars16da2uus9zrsy83h23ur42v3lglg5rmyrpqnju4/dust",
+      "osmosis_main": true,
+      "osmosis_frontier": true,
+      "pools": {
```

Comment:
We've stopped using the 'pools' property in this way, but it's good to ensure the description contains the pool id as that help us set up.